### ISSUE TITLE
Prepare website for localization

### ISSUE THREAD

**Issue by avneeshsingh** [IDPF/epubtestweb#68]
The user interface should be localizable to enable testers from different language groups to perform accessibility testing using epubtest.org.
It is long term work. But it should be in our radar.

**Comment by marisademeglio** [comment]
Noting that this also affects our test content over at https://github.com/daisy/epub-accessibility-tests

**Comment by marisademeglio** [comment]
Fixed with new website release in May 2019 (link to sources TBA)
